druid/src/main/java/org/apache/calcite/adapter/druid/DruidQuery.java

```diff
@@ -120,6 +120,12 @@ public class DruidQuery extends AbstractRelNode implements BindableRel {
           .add(new DirectOperatorConversion(SqlStdOperatorTable.SIN, "sin"))
           .add(new DirectOperatorConversion(SqlStdOperatorTable.COS, "cos"))
           .add(new DirectOperatorConversion(SqlStdOperatorTable.TAN, "tan"))
+          .add(new DirectOperatorConversion(SqlStdOperatorTable.COT, "cot"))
+          .add(new DirectOperatorConversion(SqlStdOperatorTable.ASIN, "asin"))
+          .add(new DirectOperatorConversion(SqlStdOperatorTable.ACOS, "acos"))
+          .add(new DirectOperatorConversion(SqlStdOperatorTable.ATAN, "atan"))
+          .add(new DirectOperatorConversion(SqlStdOperatorTable.ATAN2, "atan2"))
+          .add(new DirectOperatorConversion(SqlStdOperatorTable.DEGREES, "degrees"))
```

Comment:
The test cases have not covered the degrees function.

Reply:
Thanks for your reminder，had added case for degrees @NobiGo

Reply:
degrees function had some problem for druid test, so I remove first. this pr is for COT、ASIN、ACOS、ATAN、ATAN2，and change jira description.

Reply:
I'd rather leave a disabled unit test with a comment so people are aware of the problem, it's less probable that they will come check here IMO